.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+    //################################################################################
```

Comment:
Same applies here as well, copyright headers doesn't make sense and please confirm about the impact on removing .vscode directory.